### PR TITLE
Adds CMO labcoats to loadout

### PR DESCRIPTION
**Pull request by WilliamMurdoch**
Adds all 3 CMO labcoats to the loadout selection under the overwear section. Limited to just CMOs.

Let's hope I remembered how to do PRs correctly.

### CODE REVIEW

**Comment by WilliamMurdoch** [comment]
!review

**Review by Alberyk — APPROVED** [review]

File: code/modules/client/preference_setup/loadout/loadout_suit.dm [hunk 1]
```diff
@@ -93,6 +93,20 @@ datum/gear/suit/colorvest
...
+	allowed_roles = list("Chief Medical Officer")
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION
+
+/datum/gear/suit/CMOlabcoat/New()
+	..()
+	var/list/cmolabcoat = list()
+	cmolabcoat["chief medical officer labcoat"] = /obj/item/clothing/suit/storage/toggle/labcoat/cmo
+	cmolabcoat["chief medical officer labcoat alt"] = /obj/item/clothing/suit/storage/toggle/labcoat/cmoalt
+	cmolabcoat["chief medical officer labcoat alt2"] = /obj/item/clothing/suit/storage/toggle/labcoat/cmoalt2
+	gear_tweaks += new /datum/gear_tweak/path(cmolabcoat)
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_suit.dm:108]:
```suggestion
/datum/gear/suit/cmo_labcoats
	display_name = "chief medical officer labcoats selection"
	path = /obj/item/clothing/suit/storage/toggle/labcoat/cmo
	allowed_roles = list("Chief Medical Officer")
	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION

/datum/gear/suit/cmo_labcoats/New()
	..()
	var/list/cmo_labcoats = list()
	cmo_labcoats["chief medical officer labcoat"] = /obj/item/clothing/suit/storage/toggle/labcoat/cmo
	cmo_labcoats["chief medical officer labcoat, alt"] = /obj/item/clothing/suit/storage/toggle/labcoat/cmoalt
	cmo_labcoats["chief medical officer labcoat, alt 2"] = /obj/item/clothing/suit/storage/toggle/labcoat/cmoalt2
	gear_tweaks += new /datum/gear_tweak/path(cmo_labcoats)
```

**Review by SleepyGemmy — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Comment by Arrow768** [comment]
You need to merge the latest master branch into your branch

**Comment by SleepyGemmy** [comment]
see arrow's comment.

File: code/modules/client/preference_setup/loadout/loadout_suit.dm [hunk 2]
```diff
@@ -470,7 +484,8 @@ datum/gear/suit/colorvest
 
 	gear_tweaks += new /datum/gear_tweak/path(overalls)
 
+
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_suit.dm:487]:
```suggestion
```

**Pull request merged**